_content/av/the-years-that-the-days-and-months-turned-into_shafer-hall.md

```diff
@@ -0,0 +1,24 @@
+---
+title: "The Years That The Days and Months Turned Into"
+authors:
+  - "Shafer Hall"
+subcat: poetry
+external_url: "https://www.slowdownshow.org/episode/2022/05/23/680-the-years-that-the-days-and-months-turned-into"
+source_url: ""
+drive_links:
+  - "https://drive.google.com/file/d/1L1cUFxeqCq2FTsAsVSCSmbrHGTrHo9dW/view?usp=sharing"
+formats: [mp3]
+course: thought
```

Comment:
So, here we have another "secular" piece put in a Buddhist folder as a provocation... I could imagine a nice dialogue with students about the poem and "how Buddhist is it?"... But I could also see an argument against its inclusion here.  What do you think?